doc/site/en/guides/thread-primer/ipv6-addressing.md

```diff
@@ -37,8 +37,7 @@ location in the network topology.
 All devices are assigned a Router ID and a Child ID. Each Router maintains a
 table of all their Children, the combination of which uniquely identifies a
 device within the topology.  For example, consider the highlighted nodes in the
-following topology, where the number on a Router (pentagon) is the Router ID,
-and the number on an End Device (circle) is the Child ID:
+following topology, where the number in the Router (pentagon) is the Router ID, and the number in the End Device (circle) is the Child ID:
```

Comment:
Nit: Can you add a line break after "ID," where it was before?

Reply:
Of course.